drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -106,8 +106,8 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     addr_long.uint8[0] &= ~(0x01);
     addr_long.uint8[0] |=  (0x02);
     /* set short and long address */
-    at86rf2xx_set_addr_long(dev, ntohll(addr_long.uint64.u64));
-    at86rf2xx_set_addr_short(dev, ntohs(addr_long.uint16[0].u16));
+    at86rf2xx_set_addr_long(dev, &addr_long);
+    at86rf2xx_set_addr_short(dev, &addr_long.uint16[IEEE802154_LONG_ADDRESS_LEN / sizeof(uint16_t) - 1]);
```

Comment:
```suggestion
    at86rf2xx_set_addr_short(dev, &addr_long.uint16[ARRAY_SIZE(addr_long.uint16) - 1]);
```
This is easier to understand.

Reply:
Done